test/unit/factorization/mc/test_compute_t_factor.cpp

```diff
@@ -205,11 +207,6 @@ std::vector<std::tuple<SizeType, SizeType, SizeType, SizeType, SizeType, GlobalT
     {26, 39, 10, 6, 1, {1, 6}},  // k reflectors (incomplete tile)
 
     {26, 39, 10, 6, 0, {0, 6}},  // 0 reflectors
-
-    // empty matrices
-    {0, 0, 10, 6, 0, {0, 0}},
-    {39, 0, 10, 6, 0, {0, 0}},
-    {0, 26, 10, 6, 0, {0, 0}},
```

Comment:
Empty matrices do not work anymore because of SubPanelView (i.e. it is not possible to define an empty SubPanelView)

Reply:
What is the reason of the `SubPanelView` limitation?

Reply:
I opened to the possibility to define an empty Sub{Matrix/Panel}View, adapted tests and added back these test-cases for computeTFactor.

See changes in https://github.com/eth-cscs/DLA-Future/pull/456/commits/416aa0e9bf871c2acb3c0b806f4ff9500d09e2ec. Briefly, when an empty distribution is given, a SubView with invalid internal indices is created.

As soon as we agree on it, I will document it.